hardware/Makefile

```diff
@@ -92,6 +92,7 @@ vlog_defs += -DL2_BASE="32'h$(l2_base)" -DL2_SIZE="32'h$(l2_size)" -DL2_BANKS=$(
 vlog_defs += -DBOOT_ADDR="32'h$(boot_addr)" -DXPULPIMG="1'b$(xpulpimg)"
 vlog_defs += -DSNITCH_TRACE=$(snitch_trace)
 vlog_defs += -DAXI_HIER_RADIX=$(axi_hier_radix) -DAXI_MASTERS_PER_GROUP=$(axi_masters_per_group)
+vlog_defs += -DSEQ_MEM_SIZE=$(seq_mem_size) -DQUEUE_SIZE=$(queue_size)
```

Comment:
I feel those names are not very descriptive, particularly because they only apply to the CGRA version of MemPool.